go.mod

```diff
@@ -20,7 +20,6 @@ require (
 
 require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546 // indirect
```

Comment:
`go mod` is about 90% of a mystery to me.  I ran `go mod tidy`, and here we are.

Reply:
This one bites me all the time, too.  I think running `go generate ./...` re-adds this.

We should probably explicitly add this dependency somewhere in the main code, to prevent this from happening. :thinking:

Reply:
`go generate ./...` will fail and complain about attempting to use an unknown module. https://github.com/gopherjs/gopherjs/blob/master/compiler/natives/doc.go is the place which uses this package.

I think the official recommendation it to create a source `tools.go` somewhere in the source tree with something like this:

```go
//go:build ignore
package tools

import (
  _ "github.com/shurcooL/vfsgen"
)
```

The build constraint makes sure  it doesn't actually get build, but it's enough for `go mod tidy` to recognize the dependency as legit and keep it. I've been meaning to do this for a while, but haven't gotten around to it. Feel free to do this :)